### ISSUE TITLE
Release 1.5.1 in Artifact Registry?

### ISSUE THREAD

**Issue by dronenb** [GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp#443]
### TL;DR

Looking in Artifact Registry, I can see a 1.5.1 release tagged as `latest`: https://console.cloud.google.com/artifacts/docker/secretmanager-csi/us/secrets-store-csi-driver-provider-gcp/plugin/sha256:c33515ba13db6a947804238051f8d283dc072f4e91f489508a8a2d78661b7452

However, the latest release according to this repo is 1.5.0

**Expected behavior**

I would expect if there is a 1.5.1 image published in Artifact Registry that there would be a corresponding GitHub release for this project.

**Observed behavior**

Presently there is no published 1.5.1 release in this repository

### Reproduction

1.  Went to Artifact Registry repository for this project: https://console.cloud.google.com/artifacts/docker/secretmanager-csi/us/secrets-store-csi-driver-provider-gcp/plugin
2.  Noticed 1.5.1 release tagged as latest

**Screenshots**

<img width="2168" alt="Screenshot 2024-08-21 at 14 20 44" src="https://github.com/user-attachments/assets/d2bc1567-3033-431a-9f12-0e7d4d96a4ad">
